Enable specs to deploy using GitHub Actions

foolip opened the pull request:
Part of https://github.com/whatwg/meta/issues/173.

domenic commented on resources.whatwg.org/build/README.md:
```diff
@@ -12,19 +12,46 @@ The `deploy.sh` script is used by most WHATWG standards and is meant to run eith
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
```
We've been using Debian elsewhere; any particular reason to go with Ubuntu here?

foolip replied:
https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources is the set of GitHub-provided runners. We could provide our own Docker image and lock everything down, but I actually think it's not necessary for the relatively plain dependencies that we have.

foolip commented:
I fixed the shellcheck failure (missing quotes) and rebased. When I've merged this and it's deployed to resources.whatwg.org, I'll merge https://github.com/whatwg/fullscreen/pull/174 to see if I broke the existing Travis builds.
